testsuite/scripts/tutorials/test_01-lennard_jones.py

```diff
@@ -28,9 +28,13 @@
 class Tutorial(ut.TestCase):
     system = tutorial.system
 
-    def test_rdf_curve(self):
-        self.assertGreater(
-            np.corrcoef(tutorial.rdf, tutorial.theo_rdf)[1, 0], 0.985)
+    def test_rdf(self):
+        rms = np.sqrt(((tutorial.rdf - tutorial.theo_rdf)**2).mean()) / len(tutorial.r)
+        self.assertLess(rms, 5e-4)
+
+    def test_potential_energy(self):
+        # Test that the potential energy/particle agrees with the value from Verlet, Phys. Rev. 1967
+        self.assertLess(np.abs(tutorial.sim_energy + 5.38)/5.38,0.01)
```

Comment:
If I'm reading this correctly, we test `|x + 5.38| < 0.0538`, solving for `x` we get `-5.4338 < x < -5.3262`. This could be expressed as
```python
ref_energy = -5.38
self.assertAlmostEqual(tutorial.sim_energy, ref_energy, delta=ref_energy / 100)
```
or as
```python
self.assertAlmostEqual(tutorial.sim_energy, -5.38, 1)
```
because `place=1` is 0.97% if I'm not mistaken.